appsensor-core/src/main/java/org/owasp/appsensor/core/IAppsensorEntity.java

```diff
@@ -0,0 +1,14 @@
+package org.owasp.appsensor.core;
+
+import java.io.Serializable;
+
+/**
+ * Interface denoting methods required to be provided by appsensor entity objects
+ * @author Maik Jäkel(m.jaekel@xsite.de) http://www.xsite.de
+ */
+public interface IAppsensorEntity extends Serializable{
```

Comment:
Those mapping frameworks simply look for equal names. There was also the possibility to add a custom extension which would additinoally scan the objects / the json for "_id" using reflection. However, this would probably resulted in a performance a lot slower than the performance of the currently proposed solution.